src/dmd/utils.d

```diff
@@ -132,3 +132,36 @@ extern (C++) void escapePath(OutBuffer* buf, const(char)* fname)
         fname++;
     }
 }
+
+
+/**
+ * Takes a path, and escapes '(', ')', backslashes and whitespace
+ *
+ * Params:
+ *   buf = Buffer to write the escaped path to
+ *   fname = Path to escape
+ */
+extern (C++) void escapePathSpaces(OutBuffer* buf, const(char)* fname)
+{
+    while (1)
+    {
+        switch (*fname)
+        {
+        case 0:
+            return;
+        case '(':
+        case ')':
+        case '\\':
+        case ' ':
+        case '\t':
+        case '\f':
+        case '\v':
+            buf.writeByte('\\');
+            goto default;
```

Comment:
This is uncovered:

![image](https://user-images.githubusercontent.com/4370550/37889566-497fbf62-30cd-11e8-9e88-88fb502f7b16.png)

How about adding an inline unittest to cover it?

Reply:
I've added an unittest after this function that tests it for these special characters and strings without them.